scripts/tls-common.sh

```diff
@@ -95,3 +95,16 @@ if [[ -f "$MARIADB_CACERT_FILE" ]]; then
 else
     export MARIADB_TLS_ENABLED="false"
 fi
+
+configure_restart_on_certificate_update() {
+    local enabled="$1"
+    local service="$2"
+    local cert_file="$3"
+
+    if [[ "${enabled}" == "true" ]] && [[ "${RESTART_CONTAINER_CERTIFICATE_UPDATED}" == "true" ]]; then
+        python3 -m pyinotify -e IN_DELETE_SELF -v "${cert_file}" |
```

Comment:
Side note: this invocation does not make any sense to me. I don't understand why we're waiting for a file deletion, it does not look like this will be triggered the 2nd time, and it does look like inotify outputs a lot of junk that will trigger the event.

Reply:
I had in TODO list to change this as it does not work as it is, do we want to do it here or in a followup?

Reply:
let's merge this then, I'll add a follow up right after